### PR TITLE
Added Samples recorded Tests for WebApps.

### PR DESCRIPTION
**Pull request by hovsepm**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/dev/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: Samples/ResourceManagement/AppService/ManageWebAppBasic/Program.cs [hunk 1]
```diff
@@ -45,135 +37,144 @@ public static void Main(string[] args)
...
+        }
+
+        public static void RunSample(IAzure azure)
+        {
+            string app1Name = SharedSettings.RandomResourceName("webapp1-", 20);
+            string app2Name = SharedSettings.RandomResourceName("webapp2-", 20);
+            string app3Name = SharedSettings.RandomResourceName("webapp3-", 20);
+            string planName = SharedSettings.RandomResourceName("jplan_", 15);
+            string rg1Name = SharedSettings.RandomResourceName("rg1NEMV_", 24);
+            string rg2Name = SharedSettings.RandomResourceName("rg2NEMV_", 24);
```
Comment (anudeepsharma) [inline Samples/ResourceManagement/AppService/ManageWebAppBasic/Program.cs]: we can use var

File: Samples/ResourceManagement/Common/Utilities.cs [hunk 2]
```diff
@@ -22,6 +22,23 @@ namespace Microsoft.Azure.Management.Samples.Common
...
+
+        static Utilities()
+        {
+            LoggerMethod = Utilities.Log;
+        }
+
+        public static void Log(string message)
+        {
+            LoggerMethod.Invoke(message);
+        }
```
Comment (anudeepsharma) [inline Samples/ResourceManagement/Common/Utilities.cs]: Something seems missing.
LoggerMethod is Utilities.Log which is again calling LoggerMethod. are you able to print anything will this. It seems to be recursive call. Also in tests/Sample method should be assigned to something. I would say by default it should be Console.Writeline but in tests we can just override it

**Review by anudeepsharma — APPROVED** [review]
LGTM, only issue seems to be with Logger

File: Samples/ResourceManagement/Common/Utilities.cs (hunk 2 above)
Comment (hovsepm) [inline Samples/ResourceManagement/Common/Utilities.cs]: Fixed.

**Pull request merged**